media/css/firefox/new/all-test-variations-pt2.less

```diff
@@ -0,0 +1,250 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../sandstone/lib.less";
+
+// Common
+#test-primary-ctas {
+    ul {
+        list-style: none;
+    }
+
+    li {
+        display: inline-block;
+        margin: 0;
+    }
+}
+
+#test-mobile-downloads {
+    text-align: center;
+
+    li:first-child {
+        margin-right: 10px;
+    }
+
+    @media only screen and (max-width: @breakMobileLandscape) {
+        li {
+            display: list-item;
+
+            &:first-child {
+                margin-right: 0;
+            }
+        }
+    }
+}
+
+#test-direct-downloads {
+    padding: 10px 30px 0;
+    text-align: left;
+
+    li {
+        .border-box();
+        margin-bottom: 10px;
+        width: 50%;
```

Comment:
Based on conversation below, the XP/Vista link will be merged with the Windows 32-bit link, which will skirt us around this issue for now.